server/models/Profile.js

```diff
@@ -7,6 +7,12 @@ const profileSchema = new mongoose.Schema({
   lastName: {
     type: String,
   },
+  isSitter: {
+    type: Boolean,
+  },
+  rate: {
+    type: Number,
+  },
```

Comment:
Shouldn't there be a required field or a default value here?

Reply:
There some code in the auth file which creates a profile without any params which is why this bug is occurring. Ideally is this would be required by for this now this is a work around, Although I have added default.Thanks :)